css/global.css

```diff
@@ -0,0 +1,11 @@
+div.M8OgIe {
```

Comment:
Some of these are duplicates from when I was originally testing this - for now I'll leave them but eventually hope to circle back and only remove the highest parent element from the DOM.